src/evergreen/api.py

```diff
@@ -286,8 +286,9 @@ def recent_version_by_project(
         url = self._create_url(
             "/projects/{project_id}/recent_versions".format(project_id=project_id)
         )
-        version_list = self._paginate(url, params)
-        return [Version(version, self) for version in version_list]  # type: ignore[arg-type]
+        version_list = self._call_api(url, params)
+
+        return version_list.json()  # type: ignore[arg-type]
```

Comment:
this isn't the same struct expected by Version - the objects are currently returned with all-None fields